### PR TITLE
Add REPL instructions for wired fw updates

### PR DESCRIPTION
**Pull request by Hackerman342**

### CODE REVIEW

File: website/docs/contribute/website.mdx [hunk 1]
```diff
@@ -165,7 +165,7 @@ On Linux, you may find issues with the `node`, `npm`, or `yarn` version when you
 If so, re-install these packages in your venv with:
 ```bash
 nvm install node
-sudo apt install npm
+# sudo apt install npm
```
Comment (edgarriba) [inline website/docs/contribute/website.mdx:168]:
```suggestion
```

**Review by edgarriba — APPROVED** [review]
LGTM. Will be very interesting to have a short video tutorial showing this process (the simplest not including the old version stuff)

**Comment by Hackerman342** [comment]
> LGTM. Will be very interesting to have a short video tutorial showing this process (the simplest not including the old version stuff)

@edgarriba yes a video tutorial is a great idea. Captured as #85

File: website/docs/contribute/website.mdx (hunk 1 above)
Comment (Hackerman342) [reply website/docs/contribute/website.mdx:168]: Gonna keep for now in case someone else needs it

**Pull request merged**